extrabees/src/main/java/binnie/extrabees/worldgen/WorldGenHiveRock.java

```diff
@@ -21,12 +21,13 @@ public WorldGenHiveRock(int rate) {
 	@Override
 	public boolean generate(final World world, final Random random, final BlockPos pos) {
 		final IBlockState block = world.getBlockState(pos);
-		if (!block.getBlock().isAir(block, world, pos)) {
-			return true;
+		if (block.getBlock().isAir(block, world, pos)) {
+			return false;
 		}
 		if (block.getBlock().isReplaceableOreGen(block, world, pos, BlockStateMatcher.forBlock(Blocks.STONE))) {
```

Comment:
previously the code was only reached if the Block was air but then air wouldn't match for Blocks.STONE so no generation occured since b12d19c